iojs.org/aliases.json

```diff
@@ -129,5 +129,8 @@
   ] },
     { "from": "cna-discussion-list", "to": [
       "michael_dawson@ca.ibm.com"
-  ] }
+  ] },
+    { "from": "security.ecosystem@nodejs.org", "to": [
```

Comment:
Wouldn't a hyphen be more appropriate, based on other e-mail aliases?

Reply:
Also, is the @nodejs.org needed here? Other aliases don't include the domain.

Reply:
Good catch!